services/verification/README.md

```diff
@@ -36,6 +36,9 @@ interface InjectorInput {
   addresses: string[]; // For now only one address is supported
   contracts: CheckedContract[]; // Solidity and metadata files
   bytecode?: string; // If you already have bytecode of the deployed contract
+  creationData?: string;
+  constructorArguments?: string;
+  msgSender?: string;
```

Comment:
isn't it better to handle the the global variables as an object in which msgSender is one of the properties?